src/gnome-online-accounts-gtk.c

```diff
@@ -91,7 +91,11 @@ show_account (OaWindow   *window,
       goa_provider_show_account (provider,
                                  window->client,
                                  object,
+#if GTK_CHECK_VERSION(4, 15, 0)
+                                 (GtkWidget*)window,
```

Comment:
Can these be `GTK_WIDGET (window)` instead?

Reply:
Yes, that works and matches the other code style better.